__mocks__/popper.js.js

```diff
@@ -0,0 +1,12 @@
+import PopperJs from 'popper.js'
```

Comment:
dat filename 😂 

Hopefully it doesn't cause any weird... build issues

Reply:
I know... has to be the name of the module 🙃